packages/core/src/lib/select/searchable-select.svelte

```diff
@@ -282,40 +344,53 @@ const handleKeydown = createHandleKey({
         aria-selected={isSelected}
         aria-checked={isChecked}
         aria-label={isOther
-          ? [otherOptionPrefix, option].filter(Boolean).join(' ')
-          : option}
+          ? [otherOptionPrefix, optionDisplayValue(option)]
+              .filter(Boolean)
+              .join(' ')
+          : optionDisplayValue(option)}
         class={cx(
-          'flex h-7.5 w-full cursor-pointer items-center justify-start text-xs',
+          'flex cursor-pointer items-center justify-start px-2.5 py-1.5',
           multiple ? 'pl-2 pr-2.5' : 'px-2.5',
           isActive ? 'bg-light' : 'hover:bg-light'
         )}
         on:pointerdown|preventDefault
         on:mousedown|preventDefault
         on:click={() => handleSelect(option)}
-        bind:this={optionElements[option]}
+        bind:this={optionElements[option.value]}
       >
-        {#if multiple}
-          <Icon
-            cx={['mr-1 shrink-0', !isChecked && 'text-gray-6']}
-            name={isChecked ? 'checkbox-marked' : 'checkbox-blank-outline'}
-          />
-        {/if}
-        {#if isOther}
-          <Icon
-            cx="mr-1 shrink-0 text-gray-6"
-            name="plus"
-          />
-        {/if}
-        <p class="truncate whitespace-pre">
-          {#if highlight !== undefined}
-            {@const [prefix, match, suffix] = highlight}
-            {prefix}<span class="bg-yellow-100">{match}</span>{suffix}
-          {:else if isOther && otherOptionPrefix}
-            {otherOptionPrefix} {option}
-          {:else}
-            {option}
+        <div class="flex flex-row gap-2">
+          <!-- In all real cases, only one of these icons should should be active at once-->
+          <!-- (multi with icon is not a designed use case yet) -->
+          {#if multiple}
+            <Icon
+              cx={['my-0.5 shrink-0', !isChecked && 'text-gray-6']}
+              name={isChecked ? 'checkbox-marked' : 'checkbox-blank-outline'}
+            />
+          {/if}
+          {#if option.icon}
+            <Icon
+              cx={['my-0.5 shrink-0 text-gray-6']}
+              name={option.icon}
+            />
           {/if}
-        </p>
+          <div class="flex flex-col">
+            <p class="text-wrap text-sm">
+              {#if highlight !== undefined}
+                {@const [prefix, match, suffix] = highlight}
+                {prefix}<span class="bg-yellow-100">{match}</span>{suffix}
+              {:else if isOther && otherOptionPrefix}
+                {otherOptionPrefix} {optionDisplayValue(option)}
+              {:else}
+                {optionDisplayValue(option)}
+              {/if}
+            </p>
+            {#if option.description}
+              <p class="text-wrap text-xs text-subtle-2">
```

Comment:
Add an ID using `uniqueId` and attach that ID to `aria-describedby` on the named `option`

Reply:
Good idea! 23d95db